packages/components/src/internal/components/search/actions.ts

```diff
@@ -145,3 +159,119 @@ export function getProcessedSearchHits(
               })
         : undefined;
 }
+
+export function getFinderSampleTypeNames(containerFilter: Query.ContainerFilter = undefined): Promise<string[]> {
+    return new Promise((resolve, reject) => {
+        Query.executeSql({
```

Comment:
any specific reason for using `executeSql` here instead of `selectRows`? just curious as it is just a single column select without any filtering / where clause.